static/scripts/login.js

```diff
@@ -134,17 +149,29 @@ $(document).ready(() => {
 
 	$school.on('change', (event) => {
 		// due to the class 'school' being duplicated, it is necessary to listen to the element's event to get the value
+		$systems.parent().show();
 		const id = $(event.target).val();
 		const dataSystems = $(event.target).find(':selected').data('systems');
 		if (id !== '' && dataSystems) {
 			loadSystems(dataSystems);
 		} else {
 			$systems.parent().hide();
 		}
+
+		selectedSystem = dataSystems.length === 1 ? dataSystems[0].type : null;
```

Comment:
here or above select the oauth is enabled from not by type.

Reply:
I dont really understand what you mean by that

Reply:
from the system, instead of taking the type i would fetch the information if oauth is available for that system, means enabled to be used (type moodle or iserv might not give this information) additionally while a system can have oauth enabled, default (login and password) can be replaced or still be possible.